tests/ami/packages_sanity_tests.py

```diff
@@ -89,7 +89,30 @@
 from libcloud.compute.base import StorageVolume
 from libcloud.compute.base import DeploymentError
 from libcloud.compute.providers import get_driver
-from libcloud.compute.deployment import ScriptDeployment
+from libcloud.compute.deployment import (
+    ScriptDeployment,
+    FileDeployment,
+    MultiStepDeployment,
+)
+
+import libcloud.compute.base
+
+
+class ParamikoSSHClient(libcloud.compute.ssh.ParamikoSSHClient):
+    def put(self, path, contents=None, chmod=None, mode="w"):
+        result = super(ParamikoSSHClient, self).put(
+            path, contents=contents, chmod=chmod, mode=mode
+        )
+        import re
+
+        if re.match(r"^\/\w\:.*$", result):
```

Comment:
Can you please add a comment on why we need this?

And could this live upstream in Libcloud? :)

Reply:
I tried to explain this in the PR description. This is the only problem with libloud that I faced. After we `put` our deployment script into remote machine it returns remota path in unix format, and this path can not be accepted later when we execute this script.